route53_static.tf

```diff
@@ -33,6 +33,74 @@ resource "aws_ses_domain_dkim" "cyber_dhs_gov_dkim" {
   domain = aws_ses_domain_identity.cyhy_dhs_gov_identity.domain
 }
 
+# It would be nice to add aws_sns_topic_subscription resources for the
+# bounce and complaint email notifications below, but Terraform cannot
+# support that because such subscriptions must be approved out of
+# band.  See, for example, here:
+# https://www.terraform.io/docs/providers/aws/r/sns_topic_subscription.html#email
+
+# cyber.dhs.gov bounce SNS
+resource "aws_sns_topic" "cyber_dhs_gov_bounce" {
+  provider = aws.route53resourcechange
+
+  name = "${replace(aws_route53_zone.cyber_dhs_gov.name, ".", "_")}bounce"
+  tags = var.tags
+}
+resource "aws_ses_identity_notification_topic" "cyber_dhs_gov_bounce" {
+  provider = aws.route53resourcechange
+
+  identity                 = aws_ses_domain_identity.cyhy_dhs_gov_identity.domain
+  include_original_headers = true
+  notification_type        = "Bounce"
+  topic_arn                = aws_sns_topic.cyber_dhs_gov_bounce.arn
+}
+
+# cyber.dhs.gov complaint SNS
+resource "aws_sns_topic" "cyber_dhs_gov_complaint" {
+  provider = aws.route53resourcechange
+
+  name = "${replace(aws_route53_zone.cyber_dhs_gov.name, ".", "_")}complaint"
+  tags = var.tags
+}
+resource "aws_ses_identity_notification_topic" "cyber_dhs_gov_complaint" {
+  provider = aws.route53resourcechange
+
+  identity                 = aws_ses_domain_identity.cyhy_dhs_gov_identity.domain
+  include_original_headers = true
+  notification_type        = "Complaint"
+  topic_arn                = aws_sns_topic.cyber_dhs_gov_complaint.arn
+}
+
+# cyber.dhs.gov delivery SNS
+resource "aws_sns_topic" "cyber_dhs_gov_delivery" {
+  provider = aws.route53resourcechange
+
+  name = "${replace(aws_route53_zone.cyber_dhs_gov.name, ".", "_")}delivery"
+  tags = var.tags
+}
+resource "aws_ses_identity_notification_topic" "cyber_dhs_gov_delivery" {
+  provider = aws.route53resourcechange
+
+  identity                 = aws_ses_domain_identity.cyhy_dhs_gov_identity.domain
+  include_original_headers = true
+  notification_type        = "Delivery"
+  topic_arn                = aws_sns_topic.cyber_dhs_gov_delivery.arn
+}
+resource "aws_sqs_queue" "cyber_dhs_gov_delivery" {
+  provider = aws.route53resourcechange
+
+  message_retention_seconds = 1209600
```

Comment:
I always appreciate when there's a comment like `# 14 days` by numbers like these, but feel free to ignore this request.

Reply:
See commits 973c701 and 2850cbe.

Reply:
Groovy, thanks! 👍